client/components/domains/register-domain-step/index.jsx

```diff
@@ -1763,15 +1784,19 @@ class RegisterDomainStep extends Component {
 
 		const source = event.currentTarget.dataset.tracksButtonClickSource;
 
-		this.props.recordTransferDomainButtonClick( this.props.analyticsSection, source );
+		this.props.recordTransferDomainButtonClick(
+			this.props.analyticsSection,
+			source,
+			this.props.flowName
+		);
 
 		page( this.getTransferDomainUrl() );
 	};
 
 	goToUseYourDomainStep = ( event ) => {
 		event.preventDefault();
 
-		this.props.recordUseYourDomainButtonClick( this.props.analyticsSection );
+		this.props.recordUseYourDomainButtonClick( this.props.analyticsSection, this.props.flowName );
```

Comment:
```suggestion
		this.props.recordUseYourDomainButtonClick( this.props.analyticsSection, null, this.props.flowName );
```
I think the second parameter is `source` 🤔